### ISSUE TITLE
Basic plot integration

### ISSUE THREAD

**Issue by PMeira** [dss-extensions/dss_python#3]
Reproduce all plots from OpenDSS using Matplotlib and integrate them into the (DSS) `plot` command. Ideally `plot`, `show`, `visualize`, `dump` and `export` should be integrated somehow. Depending on the difficulty, other GitHub issues will be created per command.

## Files

Relevant files from the OpenDSS source tree (items in **bold** are directly related to plots)
- `Execute/ExecHelper.pas`: **`DoDI_PlotCmd`, `DoVisualizeCmd`,** `DoPropertyDump`
- `Execute/ExportOptions.pas`: `DoExportCmd`
- `Executive/PlotOptions.pas`: **`DoPlotCmd`**
- `Plot/DSSGraph.pas`
- `Plot/DSSPlot.pas`: **`TDSSPlot`**. Can implement based on `CommandLine/DSSPlot.pas`

Related but doesn't seem relevant at the moment:
- `Forms/DlgPlotOptions.pas`

## Brief description

`DoPlotCmd` processes plot options, setting up a `TDSSPlot` object and calling its `Execute` method. This indicates that we can in fact leave `DoPlotCmd` unchanged for the moment and handle the plot in the `TDSSPlot.Execute` method.
`DoVisualizeCmd` is similar, handles the parameters/options and calls `TDSSPlot.DoVisualizationPlot`. Same goes for ``, which calls `TDSSPlot.DoDI_Plot`

## Alternatives

Two clear alternatives arise:

1. Implement logic and plotting in Python.
2. Use the Pascal and only pass a list of primitives to plot.

Since the first alternative seems both easier and more flexible, it will be investigated first.

**Comment by PMeira** [comment]
On plotting, the initial implementation is available since dss_python 0.12.0. Matplotlib and SciPy are required. Some examples:

```python
from dss import DSS
import dss.plot
dss.plot.enable()

DSS.Text.Command = 'redirect EPRITestCircuits/ckt5/Run_ckt5.dss'
DSS.Text.Command = 'batchedit load..* daily=default'
DSS.Text.Command = 'Solve mode=daily'
DSS.Text.Command = 'calcincmatrix'

DSS.Text.Command = 'plot circuit'
DSS.Text.Command = 'visualize powers Transformer.mdv_sub_1'
DSS.Text.Command = 'visualize voltages Transformer.mdv_sub_1'
DSS.Text.Command = 'visualize currents Transformer.mdv_sub_1'
DSS.Text.Command = 'plot monitor object=ckt5_totalizedmonitor'
DSS.Text.Command = 'Plot Profile'
DSS.Text.Command = 'plot matrix incidence'
```

`plot circuit` will be polished for a future version.

**Comment by PMeira** [comment]
Most commands have at least an initial implementation now. Missing is `CompareCases`. I think it's safe to close this now. We only need to add some example notebooks to illustrate the feature for v1.0.
